buildpacks/ruby/Cargo.toml

```
@@ -0,0 +1,25 @@
+[package]
+name = "heroku-ruby-buildpack"
+# This crate is not published, so the only version that is used is the one in buildpack.toml.
+version = "0.0.0"
+publish = false
+edition = "2021"
+rust-version = "1.64"
+
+[dependencies]
+commons = {path = "../../commons" }
+byte-unit = "4.0.14"
+libcnb = "0.11.4"
+flate2 = "1"
+ureq = "2.5.0"
+tar = "0.4"
+serde = "1.0.145"
+url = "2.3.1"
+thiserror = "1.0.36"
+regex = "1.6.0"
+rand = "0.8.5"
+glob = { git = "https://github.com/rust-lang/glob" } # https://github.com/rust-lang/glob/issues/117
+tempfile = "3.3.0"
+
+[dev-dependencies]
+libcnb-test = ">= 0.11.4"
```

I wouldn't use a greater than range here, since it will quietly pull in breaking changes with a `Cargo.lock` refresh, whereas the version format below is equivalent to `^0.11.4` in node semver, which will pick up non-breaking changes via `Cargo.lock` refreshes, but require a Dependabot to update `Cargo.toml` to pick up the breaking change - which is what we want.

```suggestion
libcnb-test = "0.11.4"
```

---

My thinking here was that `libcnb` depends on libcnb-data which comes from `libcnb`. I don't want to accidentally run different versions of `libcnb` and `libcnb-test`. I'm a little fuzzy on the resolution rules it seems like it's not like ruby in that it allows for different versions of the same library as sub-dependencies, but according to https://stackoverflow.com/a/51722134/147390 

> cargo will attempt to find a common version which satisfies all requirements

My hope with this line is that cargo is smart enough to realize we've declared we want `libcnb` 0.11.4 which has a hard dependency on `libcnb-data` 0.11.4 and that asking for `libcnb-test` >= X will resolve to match the same version as libcnb due to the shared mutual dependency of `libcnb-data`.

Otherwise if we forget and only update one but on the other, you can run into weird type errors where it will say things like "expecting struct Store to be Store" just because they're different versions. 

I could be under/over thinking this though. I would like to hear your thoughts.

---

I updated it with your suggestion. Still curious on your thoughts above though